extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthConfig.java

```diff
@@ -31,4 +31,12 @@ public interface AuthConfig {
      */
     @WithDefault("true")
     boolean proactive();
+
+    /**
+     * Propagate security identity to support the `SecurityIdentity` CDI bean injection.
```

Comment:
@michalvavrik That would also work for `JsonWebToken`, right ?

Reply:
JWT token producer needs securityidentity therefore this enables it as well, correct